lib/elixir/lib/regex.ex

```diff
@@ -348,6 +348,9 @@ defmodule Regex do
       order. Defaults to `:first` which means captures inside the regex do not
       affect the splitting process.
 
+    * `:include_captures` - when `true`, includes in the result the matches of
```

Comment:
`run` uses capture as a verb. I am `include_captures` works better than `with_captures` for me (because of the verb).